src/main/java/com/letfy/log4j/appenders/ElasticSearchClientAppender.java

```diff
@@ -138,6 +141,13 @@ protected HttpAsyncClientBuilder configureHttpClient(HttpAsyncClientBuilder buil
             ex.printStackTrace();
         }
 
+        // Setup the date formatter
+        try {
+            formatter = DateTimeFormatter.ofPattern(getElasticDateFormat());
+        } catch (Exception ex) {
+            ex.printStackTrace();
```

Comment:
If this happens, formatter will be null and calls to `getIndexPattern` will get a null pointer exception, no? Should the exception be rethrown?